osa/scripts/calibration_pipeline.py

```diff
@@ -90,14 +91,19 @@ def calibration_sequence(drs4_pedestal_run_id: str, pedcal_run_id: str) -> int:
     return rc
 
 
-def drs4_pedestal(drs4_pedestal_run_id: str, history_file: Path) -> int:
+@trace
+def drs4_pedestal(
+    drs4_pedestal_run_id: str, pedcal_run_id: str, history_file: Path
+) -> int:
     """
     Create a DRS4 pedestal file for baseline correction.
 
     Parameters
     ----------
     drs4_pedestal_run_id : str
         String with run number of the pedestal run
+    pedcal_run_id : str
```

Comment:
why adding this argument here? Is it needed for provenance?

Reply:
Yes.
It is needed to identify link each both two calibration tasks to a single calibration workflow identified with a label of two numbers, namely `drs4_pedestal_run_id-pedcal_run_id`.